src/lightning_app/utilities/cluster_logs.py

```diff
@@ -2,34 +2,22 @@
 import queue
 from dataclasses import dataclass
 from threading import Thread
-from typing import Callable, Iterator, Optional
+from typing import Callable, Iterator, List
 
-import arrow
 import dateutil.parser
 from websocket import WebSocketApp
 
-from lightning_app.utilities.log_helpers import _error_callback, _OrderedLogEntry
+from lightning_app.utilities.log_helpers import _OrderedLogEntry
 from lightning_app.utilities.logs_socket_api import _ClusterLogsSocketAPI
-from lightning_app.utilities.network import LightningClient
+from pytorch_lightning.utilities import AttributeDict
```

Comment:
This is a dependency on the `pytorch_lightning` package, which means we cannot use the app package separately.

Reply:
@Borda @tchaton is this fine ?

Reply:
Ohh...good point! 

What do you recommend I do in this case?

Reply:
@nmiculinic for now, we shouldn't use anything form `pytorch_lightning` maybe just duplicate it in the app utilities.

Reply:
@tchaton what do you recommend to use? Maybe copy&paste the AttributeDict from the pytorch lightning?

Reply:
Copy the code to App or move the class to utilities...
cc: @carmocca

Reply:
Yes, it could go into https://github.com/Lightning-AI/utilities/tree/main/src/lightning_utilities/core